pandas/core/tools/numeric.py

```diff
@@ -43,6 +43,10 @@ def to_numeric(arg, errors='raise', downcast=None):
         checked satisfy that specification, no downcasting will be
         performed on the data.
 
+        Also Note that the default return dtype is `float64` or `int64`
+        depending on the data supplied.`downcast` for `float` or `int`
+        should only be used if output desired is `float32` or `int32`
```

Comment:
I would maybe add something like this to the "extended summary" (just after the first line of the docstring), so something like:

```
Convert argument to a numeric type.

The default return dtype is `float64` or `int64` depending on the data supplied. 
Use the `downcast` parameter to obtain other dtypes.

Parameters
-----------
...
```

Reply:
@jorisvandenbossche  Sure, I will modify the PR asap